athena-db2-as400/src/test/java/com/amazonaws/athena/connectors/db2as400/Db2As400CompositeHandlerTest.java

```diff
@@ -0,0 +1,57 @@
+/*-
+ * #%L
+ * athena-db2-as400
+ * %%
+ * Copyright (C) 2019 - 2022 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.db2as400;
+
+import com.amazonaws.athena.connectors.jdbc.connection.DatabaseConnectionConfig;
+import com.amazonaws.athena.connectors.jdbc.manager.JDBCUtil;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(JDBCUtil.class)
+@PowerMockIgnore({"com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*",
+        "javax.management.*","org.w3c.*","javax.net.ssl.*","sun.security.*","jdk.internal.reflect.*"})
+public class Db2As400CompositeHandlerTest
+{
+    private Db2As400CompositeHandler db2As400CompositeHandler;
+    private static final Logger logger = LoggerFactory.getLogger(Db2As400CompositeHandler.class);
+
+    static {
+        System.setProperty("aws.region", "us-east-1");
+    }
+
+    @Test
+    public void Db2CompositeHandlerTest() {
+        DatabaseConnectionConfig databaseConnectionConfig = new DatabaseConnectionConfig("testCatalog1", Db2As400Constants.NAME,
+                "dbtwo-iseries://jdbc:as400://testhost:user=dummy;password=dummy;");
```

Comment:
See this link for what I mean: https://regex101.com/r/pgOPVW/1

Alternatively if we really do want to support hyphens in addition to numbers, we can just make the regex like this:
```
private static final String CONNECTION_STRING_REGEX = "([a-zA-Z0-9\-]+)://(.*)";
```

Reply:
The test classes does not reflect the pattern yet, we were going to update it.

Reply:
Can we go ahead and update the regex pattern/ please let us know.

Reply:
Yeah go ahead, but without the hyphen, then just use `db2as400://` instead of my comments about `db2-as400://`

So to be clear, the regex should be
```
private static final String CONNECTION_STRING_REGEX = "([a-zA-Z0-9]+)://(.*)";
```

Reply:
sure.